cli/kubernetes/installer.go

```diff
@@ -83,17 +81,26 @@ func (i *Installer) PopulateNeededOptions(reader OptionsReader) error {
 
 // ShowNeededOptions prints the options and their values to stdout, to
 // inform the user of the detected and chosen configuration
-func (i *Installer) ShowNeededOptions() {
-	fmt.Println("Configuration...")
+func (i *Installer) ShowNeededOptions(ui *ui.UI) {
```

Comment:
same things as with the Deployments. The UI could be an attribute of the Installer so it's available in all methods.

Reply:
See above, rejected due to an import circle. Will re-check if still circular.

Reply:
Ok. Falsified. The circular import is gone.
Will add an UI field to installer and reduce usage in function signatures.

Reply:
Done with commit a3f7aded87a42a0a1ac2074c66bdac1e5414f43a.
Looking into `Deployment` now. That also gives me the time to have the back of my mind mull more on the merging of `Installer` and `CarrierClient` variant.